### DISCUSSION TITLE
webm support for video attachments in QA issues

### DISCUSSION THREAD

**Discussion by hholst80** [community/community#10814] [Projects and Issues]
The default video format for Slack is webm and it would be very convenient to be able to attach those video snippets in a GitHub issue.

**Comment by TheRadicalDreamer** [comment]
Same as https://github.com/github/feedback/discussions/8864 it does support it already.

But the upload code filters it. Which is really silly.

If you rename your file to .mp4 you will be able to upload it.  :/
